core/src/test/kotlin/com/google/container/tools/core/settings/UsageTrackerConfigurableTest.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.core.settings
+
+import com.google.common.truth.Truth
+import com.google.container.tools.core.analytics.UsageTrackerManagerService
+import com.google.container.tools.test.ContainerToolsRule
+import com.google.container.tools.test.TestService
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ * Tests for [UsageTrackerConfigurable]
+ */
+class UsageTrackerConfigurableTest {
+    @get:Rule
+    val containerToolsRule = ContainerToolsRule(this)
+
+    @TestService
+    @MockK
+    private lateinit var usageTrackerManagerService: UsageTrackerManagerService
+
+    private lateinit var usageTrackerConfigurable: UsageTrackerConfigurable
+
+    @Before
+    fun setUp() {
+        usageTrackerConfigurable = UsageTrackerConfigurable()
+    }
+
+    @Test
```

Comment:
since UI state is modified here, it makes sense to use `@UITest` as well to protect against possible racing